CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5.0)
+cmake_minimum_required(VERSION 3.24.2)
```

Comment:
I suggest not to update the version of the CMake to a too recent version, instead specific policies could be enabled in either DCMQI or in the client project.